### PR TITLE
Add simple stats table

### PR DESCRIPTION
**Pull request by atomgomba**
Display a table of extreme values under the values table. Show stats like min/max sticks position, min/max altitude, etc. I'm using this to quickly evaluate the quality of data from test flights.

### CODE REVIEW

File: js/main.js
Comment (haslinghuis) [inline js/main.js:176]: :space_invader:

**Comment by nerdCopter** [comment]
screenshot?

**Comment by atomgomba** [comment]
> screenshot?

image

File: js/main.js
Comment (haslinghuis) [reply js/main.js:176]: Trailing space

File: js/main.js
Comment (haslinghuis) [inline js/main.js:235]: :space_invader:

File: js/simple-stats.js
Comment (haslinghuis) [inline js/simple-stats.js:6]: Please use strict equality https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality

**Comment by nerdCopter** [comment]
what are the requirements?  may need some error-checking
image
image

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by atomgomba** [comment]
> what are the requirements? may need some error-checking image image

Good catch! I didn't do testing using logs without GPS data, classic. Pushed the fix, it now should just skip fields with missing data.

**Comment by nerdCopter** [comment]
no crashes on many logs. here is sample of 4.
image

**Review by nerdCopter — APPROVED** [review]

File: js/simple-stats.js
Comment (haslinghuis) [inline js/simple-stats.js:6]: Potential bug was fixed in https://github.com/betaflight/betaflight/pull/11911

File: js/simple-stats.js
Comment (atomgomba) [reply js/simple-stats.js]: Thank you for the heads up! I'm going to include request from #309 too :ok_hand:

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
